build/setup.d/40-nvidia.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# See https://github.com/NVIDIA/nvidia-docker/wiki/Deploy-on-Amazon-EC2
+# But we're using the run script to not depend on build tools
+DRIVER_ARCH="Linux-x86_64"
+DRIVER_VERSION="375.26"
+DRIVER_FILENAME="NVIDIA-${DRIVER_ARCH}-${DRIVER_VERSION}.run"
+DRIVER_CHECKSUM="d60819b2e377398c7296999ab5e7c1a4"
+
+DOCKER_DRIVER_VERSION="1.0.0"
```

Comment:
I see there is a 1.1.0 version out that mentions some bugfixes: https://github.com/NVIDIA/nvidia-docker/releases

Should we upgrade to it here too?